test/test_mysql2.rb

```diff
@@ -0,0 +1,148 @@
+require 'test/unit'
+require 'semian/mysql2'
+require 'toxiproxy'
+require 'timecop'
+
+class TestMysql2 < Test::Unit::TestCase
+  def setup
+    @proxy = Toxiproxy[:semian_test_mysql]
+    Semian.destroy(:mysql_testing)
+  end
+
+  def teardown
+    threads.each { |t| t.kill }
+    @threads = []
```

Comment:
I'd use an `attr_accessor` here. It's awkward to use instance variable + method.